src/main/java/com/faforever/client/chat/ChannelTabController.java

```diff
@@ -183,6 +205,13 @@ private void updateChannelTopic() {
             }
           });
     }
+    topicTextField.setVisible(false);
+    topicPane.setDisable(false);
+    if (topicTriggeredByUser) {
+      topicTriggeredByUser = false;
+      String username = userService.getUsername();
+      chatService.sendMessageInBackground(channelName, username + " updated the channel topic");
```

Comment:
Why are we manually sending a notice that the topic has changed? Doesn't the irc server do this

Reply:
It sometimes happens that players do not notice that the topic has been changed when they are in a different place (lobby, during the game). Especially when the TD often changes the topic

Reply:
We cannot know that the joined channel has an already established topic or not.

Reply:
Hmm.. I think I understand what you mean. I'll change it.